cmd/fetchd/cmd/gen_asi_upgrade_manifest.go

```diff
@@ -0,0 +1,43 @@
+package cmd
+
+import (
+	"encoding/json"
+	"fmt"
+	"github.com/cosmos/cosmos-sdk/types"
+	"os"
+)
+
+type ASIUpgradeTransfer struct {
+	From   string      `json:"from"`
+	Amount types.Coins `json:"amount"`
+}
+
+type ASIUpgradeTransfers struct {
+	Transfer []ASIUpgradeTransfer `json:"transfer"`
+	To       string               `json:"to"`
+}
+
+type ASIUpgradeManifest struct {
+	IBC            *ASIUpgradeTransfers `json:"ibc"`
+	Reconciliation *ASIUpgradeTransfers `json:"reconciliation"`
+}
+
+func SaveASIManifest(manifest *ASIUpgradeManifest) error {
+	var serialised_manifest []byte
+	var err error
+	if serialised_manifest, err = json.MarshalIndent(manifest, "", "\t"); err != nil {
+		return fmt.Errorf("failed to marshal manifest: %w", err)
+	}
+
+	var f *os.File
+	const manifestFilePath = "asi_upgrade_manifest.json"
```

Comment:
This needs to be changed: The location of the file needs to be specified. The `~/.fetchd/config/` loaction is probably the best (the same as `genesis.json`)